### DISCUSSION TITLE
en-GB transcribed in American English instead of British English

### DISCUSSION THREAD

**Discussion by anth12** [deepgram/community#749] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT

### Details

Generated text always uses American English.

Despite using `language=en-GB`, words such as colour & neighbour are miss-transcribed using Amarican English (color, neighbor).

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?language=en-GB&model=nova-2&smart_format=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

ed70c549-117a-45cc-8ee5-d45185ad1d34

### If possible, please attach your code or paste it into the text box.

Test requests generated from the playground:
https://playground.deepgram.com/?endpoint=listen&smart_format=true&language=en-GB&model=nova-2

```
fetch("https://api.deepgram.com/v1/listen?language=en-GB&model=nova-2&smart_format=true", {
  "headers": {
    "authorization": "token ...",
    "content-type": "application/json"
  },
  "body": "{\"url\":\"https://manage.deepgram.com/storage/assets/3deeadb2c938ef7d7ebacb989be65c96dfbf88ba\"}",
  "method": "POST"
});
```

### If possible, please attach an example audio file to reproduce the issue.

[80001_GB_ENGLISH_SAMPLE5_Genny Long_20230605_Remote Work.zip](https://github.com/deepgram/community/files/15249194/80001_GB_ENGLISH_SAMPLE5_Genny.Long_20230605_Remote.Work.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@anth12  Have you tried this exact audio using our [Nova-1](https://developers.deepgram.com/docs/keywords) model?  I'm curious if there is difference between the 2 GB languages models for Nova 2 vs Nova 1.

`https://api.deepgram.com/v1/listen?language=en-GB&model=nova&smart_format=true`

**Reply by anth12** [reply]
Hi @jpvajda,
I've just tried with Nova and got the same American spellings.
https://api.deepgram.com/v1/listen?language=en-GB&model=nova&smart_format=true

**Reply by lukeocodes** [reply, marked as answer]
As a Brit on the DG team I can sympathise here. For personal projects, I have used open source packages that replace words. Not ideal, but should unblock you while we raise this internally.
